src/rpdk/core/test.py

```diff
@@ -201,6 +201,7 @@ def invoke_test(args, project, overrides, inputs):
             overrides,
             inputs,
             args.role_arn,
+            args.enforce_timeout,
```

Comment:
Does this flag require an int value as argument? Can it default to 60seconds if you supply only the flag.

Also, can we include an update to docs/README with this change?

Reply:
This takes in a string and converts it to int. Right now if no flag is provided it defaults to 60/30 seconds depending on the handler. I can add the feature to default to 60/30 with no flag value. 

I will add readme for this.

Reply:
Oh I thought this PR was to make 'no timeout enforced' the default - because it fails all the time on SAM Local?

Reply:
it just allows customers to increase the timeout. If no --enforce-timeout arg is mentioned it assert on 60 seconds for CUD and 30 seconds fo RL handlers.